### PR TITLE
Polish LambdaHttpHandler instances

### PR DESCRIPTION
**Pull request by geoand**
This came about while trying to reproduce the
problem mentioned in #21559

### CODE REVIEW

File: extensions/amazon-lambda-http/runtime/src/main/java/io/quarkus/amazon/lambda/http/LambdaHttpHandler.java [hunk 1]
```diff
@@ -46,7 +46,7 @@
 
     private static final int BUFFER_SIZE = 8096;
 
-    private static Headers errorHeaders = new Headers();
+    private static final Headers errorHeaders = new Headers();
```
Comment (gsmet) [inline extensions/amazon-lambda-http/runtime/src/main/java/io/quarkus/amazon/lambda/http/LambdaHttpHandler.java:49]: I think code checkers will complain about the case as it's a constant (same for the other file).

**Review by gsmet — APPROVED** [review]

File: extensions/amazon-lambda-http/runtime/src/main/java/io/quarkus/amazon/lambda/http/LambdaHttpHandler.java (hunk 1 above)
Comment (geoand) [reply extensions/amazon-lambda-http/runtime/src/main/java/io/quarkus/amazon/lambda/http/LambdaHttpHandler.java:49]: I don't get it, what would they complain about?

Comment (gsmet) [reply extensions/amazon-lambda-http/runtime/src/main/java/io/quarkus/amazon/lambda/http/LambdaHttpHandler.java:49]: Case for constants should be `LIKE_THIS`.

Comment (geoand) [reply extensions/amazon-lambda-http/runtime/src/main/java/io/quarkus/amazon/lambda/http/LambdaHttpHandler.java:49]: Ah, the case. You mentioned it, but it just didn't register..

**Pull request merged**